app/presenters/atom_presenter.rb

```diff
@@ -0,0 +1,22 @@
+class AtomPresenter
+
+  def initialize(finder)
+    @finder = finder
+  end
+
+  def title
+    finder.name
+  end
+
+  def entries
+    @finder.results.documents
```

Comment:
This can just be `finder` as you've gone to the trouble of setting `attr_reader :finder`.